### DISCUSSION TITLE
Log in to Microsoft Store from WinGet

### DISCUSSION THREAD

**Discussion by sharpninja** [microsoft/winget-cli#1616] [Q&A] [answered]
Is there a way to log into the MS Store using my Microsoft Credentials with winget for the purpose of obtaining a license for a locally downloaded appx?

**Comment by jedieaston** [comment, marked as answer]
There is not, and I don't think it can acquire a license on its own anyway. 

(If you want to log into the Microsoft Store, you have to do it through the GUI, although this is [changing](https://github.com/microsoft/winget-cli/issues/1585) for free apps)
